samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/PublishedAssessmentFacadeQueriesAPI.java

```diff
@@ -276,6 +276,12 @@ public ArrayList getBasicInfoOfLastSubmittedAssessments(String agentId,
     public HashMap getTotalSubmissionPerAssessment(String agentId, String siteId);
 
   public Integer getTotalSubmission(String agentId, Long publishedAssessmentId);
+  
+  /**
+   * return submission numbers for the assessment by giving the publishedAssessmentId
+   * Added by -Qu for assessment deletion safe check SAM-2587
+   */
```

Comment:
Can you make this JavaDoc (and all others added) real JavaDocs, with @params, @returns, etc.

Also, it is not necessary to state your name in code (-Qu), as the commits will be attributed to you and the author(s) of any line of code can be determined with git/svn blame.